### PR TITLE
Update CVU to no longer to QDM calculation.

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into the Cypress Validation Utility require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR:
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: app/models/qrda_file.rb [hunk 1]
```diff
@@ -49,8 +49,8 @@ def perform_validatons(validation_errors)
       errs = v.validate(content, :file_name => @filename)
       errs.each do |e|
         e.validator ||= v.class.name
-        # make cat1 validation error types a warning for data criteria outside the measure definition
-        if (e.validator == 'HealthDataStandards::Validate::DataValidator') && (e.message.include? 'data criteria outside')
+        # make cat1 validation error types a warning for data criteria outside the measure definition'
```
[the remark this answers is not in the document]
Comment (dczulada) [reply app/models/qrda_file.rb]: my guess is I put a bye bug in there when testing, and when I removed it the ' was added to the line above as I was deleting it.

File: config/mongoid.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ development:
 test:
   clients:
     default:
-      database: cypress_validation_utility_test
```
[the remark this answers is not in the document]
Comment (dczulada) [reply config/mongoid.yml]: to work with the current configuration of js-ecqm-engine

File: lib/cypress/go_importer.rb [hunk 3]
```diff
@@ -2,15 +2,18 @@
 
 module Cypress
```
[the remark this answers is not in the document]
Comment (dczulada) [reply lib/cypress/go_importer.rb]: This is a bad naming convention that found its way in to Cypress v4.  These are really just post-processing steps after import

**Comment by okeefm** [comment]
Getting an issue when I upload a QRDA .zip:

Screen Shot 2019-04-26 at 8 48 09 AM

Settings I used:

Screen Shot 2019-04-26 at 8 48 57 AM

File: config/mongoid.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (okeefm) [reply config/mongoid.yml]: Makes sense.

**Comment by okeefm** [comment]
Just needs the update to Nokogiri/HDS applied.
